### PR TITLE
docs: add transaction management how-to and related new terms

### PR DESCRIPTION
**Pull request by linnnsss**
* add new how-to under how-tos
* add RBF, CPFP, fee bumping under glossary

### CODE REVIEW

File: website/docs/how-tos/how-to-manage-txs.mdx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+<img
+  alt="Transition Between Different Transaction States"
+  src={useBaseUrl("img/how-tos/transaction-states.png")}
+  width="688"
+/>
+
+## Pending Transaction
+
+The newly created transaction enters the **Pending** state.
```
Comment (yfeng2824) [inline website/docs/how-tos/how-to-manage-txs.mdx:41]:
```suggestion
A newly created transaction starts in the **Pending** state.
```

File: website/docs/how-tos/how-to-manage-txs.mdx [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+The generator must store pending transactions locally and regularly send them to CKB nodes. It's essential because CKB nodes may drop transactions from their pools.
+
+It is not recommended for the generator to use output Cells of pending transactions unless the transaction is stuck and requires a <Tooltip>fee bumping</Tooltip>. If chaining pending transactions is necessary, limit the chain’s length.
```
Comment (yfeng2824) [inline website/docs/how-tos/how-to-manage-txs.mdx:45]:
```suggestion
It is not recommended for the generator to use output Cells of pending transactions unless the transaction is stuck and requires a <Tooltip>fee bumping</Tooltip>. If chaining pending transactions is unavoidable, limit the chain’s length.
```

File: website/docs/how-tos/how-to-manage-txs.mdx [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+Once the transaction appears in a block on the canonical chain, it turns to **Confirming**.
```
Comment (yfeng2824) [inline website/docs/how-tos/how-to-manage-txs.mdx:47]:
```suggestion
The transaction turns to the **Confirming** state once it appears in a block on the canonical chain.
```

File: website/sidebars.js [hunk 4]
```diff
@@ -116,6 +116,7 @@ export default {
         "how-tos/how-to-calculate-tx-hash",
         "how-tos/how-to-calculate-script-hash",
         "how-tos/how-to-calculate-code-hash",
+        "how-tos/how-to-manage-txs",
```
Comment (RetricSu) [inline website/sidebars.js:119]: can you move this under "how-tos/how-to-calculate-tx-hash" ?

**Pull request merged**